libs/langchain/langchain/schema/runnable/base.py

```diff
@@ -2296,6 +2296,25 @@ class RunnableBinding(RunnableSerializable[Input, Output]):
     class Config:
         arbitrary_types_allowed = True
 
+    def __init__(
+        self,
+        *,
+        bound: Runnable[Input, Output],
+        kwargs: Mapping[str, Any],
+        config: Optional[Mapping[str, Any]] = None,
+        **other_kwargs: Any,
+    ) -> None:
+        config = config or {}
```

Comment:
A doc-string could help here... this is a bit "non-local" -- i.e., would be difficult for another developer why this validation code makes sense here